core/deployment/src/main/java/io/quarkus/deployment/index/ApplicationArchiveBuildStep.java

```diff
@@ -213,24 +215,32 @@ private static void addMarkerFilePaths(Set<String> applicationArchiveFiles,
             }
             final PathCollection artifactPaths = dep.getResolvedPaths();
             boolean containsMarker = false;
+            FileSystem fs = null;
             for (Path p : artifactPaths) {
                 if (root.isExcludedFromIndexing(p)) {
                     continue;
                 }
-                if (Files.isDirectory(p)) {
+                if (dep.isWorkspacetModule()) {
```

Comment:
Actually those failures look related, and my guess is that it would be because of this change, not 100% sure why though.

Reply:
The gradle tests fail at `MultiModuleIncludedBuildTest`. This does seem related.

Thank you for the pointer!

Reply:
Done